Source/Initialization/WarpXInitData.cpp

```diff
@@ -25,6 +25,40 @@
 
 using namespace amrex;
 
+void
+WarpX::PostProcessBaseGrids (BoxArray& ba0) const
+{
+    if (numprocs != 0) {
+        const Box& dom = Geom(0).Domain();
+        const IntVect& domlo = dom.smallEnd();
+        const IntVect& domlen = dom.size();
+        const IntVect sz = domlen / numprocs;
+        const IntVect extra = domlen - sz*numprocs;
+        BoxList bl;
+#if (AMREX_SPACEDIM == 3)
```

Comment:
Could you add a line of comments to explain what you do with the `extra` cells below?

Reply:
Added some comments.